src/components/pages/Categories/Categories.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div
+    class="categoryObject"
+  >
+    <app-category-post
+      class="categoryPost"
```

Comment:
[imo]

以下のようにした方が、Sass で表現した時に可読性が上がると思います。
また、他のファイルの記述を見ても以下のようにケバブケースになっていそうなので修正をお願いします。

```suggestion
      class="category-post"
```

Reply:
class名をキャメルケースからケバブケースの記述に変更しました。